### PR TITLE
Support Storybook 6

### PR DESCRIPTION
**Pull request by daybrush**
#12  #15

### CODE REVIEW

**Review by mixed — APPROVED** [review]
awesome!

File: README.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
 
 [![npm version](https://img.shields.io/npm/v/storybook-addon-preview.svg?style=flat-square&color=007acc&label=Version)](https://badge.fury.io/js/storybook-addon-preview)
 
-Storybook Addon Preview can show user selected [knobs](https://github.com/storybookjs/storybook/tree/master/addons/knobs) in various framework code in [Storybook](https://storybook.js.org/)
+Storybook Addon Preview can show user selected [args(controls)](https://github.com/storybookjs/storybook/tree/next/addons/controls) or [knobs](https://github.com/storybookjs/storybook/tree/master/addons/knobs) in various framework code in [Storybook](https://storybook.js.org/)
```
Comment (WoodNeck) [inline README.md:5]: Why don't you use the master branch's document?
I checked it and it is identical to the next branch's
https://github.com/storybookjs/storybook/tree/master/addons/controls

File: README.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
 
 
 
 [![](./images/screenshot.png)](https://naver.github.io/egjs-infinitegrid/storybook/)
 
 ## Getting Started
-First of all, you need to install Knobs into your project as a dev dependency.
+* Support for Storybook 6.
```
Comment (WoodNeck) [inline README.md:12]: This looks very weird.
I think a sentence like "Storybook 6 or newer is required." is more appropriate in this case.

File: README.md [hunk 3]
```diff
@@ -25,6 +26,74 @@ import "storybook-addon-preview/register";
 
 Now, write your stories with preview.
 
+### How to use with args(controls)
+* [Install @storybook/addon-args](https://github.com/storybookjs/storybook/tree/next/addons/controls)
```
Comment (WoodNeck) [inline README.md:30]: Also using next branch here

**Review by WoodNeck — APPROVED** [review]
LGTM

**Pull request merged**